PalasoUIWindowsForms/SettingProtection/SettingsProtection.cs

```diff
@@ -62,8 +63,27 @@ public static SettingsProtectionSettings Settings
 
 		public static string FactoryPassword
 		{
-			get { return  (Application.ProductName.Substring(0, 1) + "7" + Application.ProductName.Substring(1, Application.ProductName.Length-1)).ToLower(); }
+			get
+			{
+				var productName = PreferredProductName;
+				return  (productName.Substring(0, 1) + "7" + productName.Substring(1, productName.Length-1)).ToLower();
```

Comment:
The second argument could be omitted from the second Substring call - this would make it easier to read:

    return  (productName.Substring(0, 1) + "7" + productName.Substring(1).ToLower();